server/src/main/java/org/apache/uniffle/server/ShuffleServerConf.java

```diff
@@ -538,6 +538,19 @@ public class ShuffleServerConf extends RssBaseConf {
                   + "network_bandwidth = 10Gbps, buffer size should be ~ 1.25MB."
                   + "Default is 0, OS will dynamically adjust the buf size.");
 
+  public static final ConfigOption<Integer> TOP_N_SHUFFLE_DATA_SIZE_OF_APP_LEVEL =
+      ConfigOptions.key("rss.server.topN.shuffle.data.size.of.app-level")
```

Comment:
How about `rss.server.topN.appShuffleDataSize.number` ?

Reply:
How about this refer to: https://github.com/apache/incubator-uniffle/pull/1363#discussion_r1425108117

`rss.server.topN.app-shuffle-data-size.number`